approvaltests/reporters/diff_reporter.py

```diff
@@ -23,6 +23,6 @@ def __init__(self, reporter_factory=None):
 
         reporters = list(factory.get_all_reporters_from_config())
         reporters.extend(
-            [ReportWithVSCode(), ReportWithDiffCommandLine(), PythonNativeReporter()]
+            [ReportWithVSCode(), ReportWithVSCodeMacOS(), ReportWithDiffCommandLine(), PythonNativeReporter()]
```

Comment:
This should break a test I think...
Have you run the tests?

Reply:
I think I've found the breaking test and updated the approval